Formula/orientdb.rb

```diff
@@ -27,9 +26,16 @@ def install
     chmod 0755, Dir["bin/*"]
     libexec.install Dir["*"]
 
-    mkpath "#{libexec}/log"
-    touch "#{libexec}/log/orientdb.err"
-    touch "#{libexec}/log/orientdb.log"
+    (var/"db/orientdb").mkpath
+    (var/"log/orientdb").mkpath
+    (var/"run/orientdb").mkpath
+    touch "#{var}/log/orientdb/orientdb.err"
+    touch "#{var}/log/orientdb/orientdb.log"
+    inreplace "#{libexec}/config/orientdb-server-config.xml", "</properties>", "  <entry name=\"server.database.path\" value=\"#{var}/db/orientdb\" />\n    </properties>"
```

Comment:
Use `EOS.undent` instead of `\n`s. Split this into multiple lines so it wraps at ~80 characters.